common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java

```diff
@@ -258,32 +263,33 @@ void failsForInvalidVisitedCountries(String visitedCountries) {
   }
 
   @ParameterizedTest
-  @ValueSource(ints = { DiagnosisKey.MIN_TRANSMISSION_RISK_LEVEL - 1, DiagnosisKey.MAX_TRANSMISSION_RISK_LEVEL + 1 })
+  @ValueSource(ints = { MIN_TRANSMISSION_RISK_LEVEL - 1, MAX_TRANSMISSION_RISK_LEVEL + 1 })
   void transmissionRiskLevelMustBeInRange(int invalidRiskLevel) {
     assertThat(catchThrowable(() -> keyWithRiskLevel(invalidRiskLevel)))
         .isInstanceOf(InvalidDiagnosisKeyException.class)
         .hasMessage(
-            "[Risk level must be between " + DiagnosisKey.MIN_TRANSMISSION_RISK_LEVEL + " and "
-                + DiagnosisKey.MAX_TRANSMISSION_RISK_LEVEL + ". Invalid Value: " + invalidRiskLevel + "]");
+            "[Risk level must be between " + MIN_TRANSMISSION_RISK_LEVEL + " and "
+                + MAX_TRANSMISSION_RISK_LEVEL + ". Invalid Value: " + invalidRiskLevel + "]");
   }
 
   @ParameterizedTest
-  @ValueSource(ints = { DiagnosisKey.MIN_TRANSMISSION_RISK_LEVEL, DiagnosisKey.MAX_TRANSMISSION_RISK_LEVEL })
+  @ValueSource(ints = { MIN_TRANSMISSION_RISK_LEVEL, MAX_TRANSMISSION_RISK_LEVEL })
   void transmissionRiskLevelDoesNotThrowForValid(int validRiskLevel) {
     assertThatCode(() -> keyWithRiskLevel(validRiskLevel)).doesNotThrowAnyException();
   }
 
   @ParameterizedTest
-  @ValueSource(ints = { -15, -17, 4001 })
+  @ValueSource(ints = { MIN_DAYS_SINCE_ONSET_OF_SYMPTOMS - 1, -17, 4001 })
```

Comment:
Why do we have here MIN_DAYS_SINCE_ONSET_OF_SYMPTOMS - 1?

Reply:
because it's a negative test: testFailureInvalidDSOS